lms/djangoapps/discussion/rest_api/discussions_notifications.py

```diff
@@ -62,6 +59,32 @@ def _send_notification(self, user_ids, notification_type, extra_context=None):
         )
         USER_NOTIFICATION_REQUESTED.send_event(notification_data=notification_data)
 
+    def _send_course_wide_notification(self, course_id, notification_type, audience_filters=None, extra_context=None):
```

Comment:
we don't need to pass `course_id` as parameter. This is already available as `self.course.id`